scripts/prepare_gene_ann.py

```diff
@@ -49,7 +49,7 @@ def main():
     res['chr'] = res['chr'].apply(lambda x: x.replace('chr', ''))
     res['chr'] = res['chr'].apply(lambda x: '23' if x == 'X' else x)
     res['chr'] = res['chr'].apply(lambda x: '24' if x == 'Y' else x)
-    res['chr'] = res['chr'].apply(lambda x: '25' if x == 'M' else x)
+    res['chr'] = res['chr'].apply(lambda x: '26' if x == 'M' else x)
```

Comment:
Was this change to 26 intentional?

Reply:
Yes, I changed it since, if I understood it correctly from the naming convention, in FG data 26 is used for chrM (e.g.  gs://finngen-production-library-red/finngen_R12/chipd_1.0/data/vcf/r12_axiom_chr26_subset.vcf.gz). Is that correct or did I misunderstand something @Fedja ?